CHANGELOG/CHANGELOG-2.1.md

```diff
@@ -0,0 +1,72 @@
+# Changelog 2.1
+
+## 2.1.0 - 2024-07-26
+
+### Added
+
+- Add a new feature `BGPPolicy` that allows users to run a BGP process on selected Kubernetes Nodes and advertise Service IPs, Pod IPs, and Egress IPs to remote BGP peers.
+  - Add `BGPPolicy` API and Controller. ([#6009](https://github.com/antrea-io/antrea/pull/6009) [#6203](https://github.com/antrea-io/antrea/pull/6203), [@hongliangl])
+  - Add BGP datapath interface and implement goBGP integration. ([#6447](https://github.com/antrea-io/antrea/pull/6447), [@hongliangl])
+  - Add documentation for the `BGPPolicy` feature. ([#6524](https://github.com/antrea-io/antrea/pull/6524), [@hongliangl])
+  - Refer to [this document](https://github.com/antrea-io/antrea/blob/release-2.1/docs/bgp-policy.md) for more information about this feature.
+- Add a new feature `NodeLatencyMonitor` that allow users to do east/west connectivity monitoring and provides an API to query inter-Node latency.
+  - Add `NodeLatencyMonitor` feature in Antrea Agent that performs ping mesh and monitor the latency between Nodes. ([#6120](https://github.com/antrea-io/antrea/pull/6120), [@IRONICBo])
+  - Export Node latency query API definition. ([#6392](https://github.com/antrea-io/antrea/pull/6392) [#6479](https://github.com/antrea-io/antrea/pull/6479), [@IRONICBo])
```

Comment:
I think it would be ok to drop these 2 sub-items, and just reference all 3 PRs in the main list item

Reply:
Merged into the main item.